src/dbsync/api.rs

```diff
@@ -23,10 +23,8 @@ pub fn get_utxo_tokens(
     let multi_assets = multi_asset::table
         .inner_join(ma_tx_out::table.on(multi_asset::id.eq(ma_tx_out::ident)))
         .inner_join(tx_out::table.on(tx_out::id.eq(ma_tx_out::tx_out_id)))
-        .inner_join(utxo_view::table.on(utxo_view::id.eq(tx_out::id)))
-        .filter(utxo_view::tx_id.eq(tx_id))
-        .filter(utxo_view::index.eq(tx_index))
-        //.select((multi_asset::id,multi_asset::policy,multi_asset::name,multi_asset::fingerprint))
+        .filter(tx_out::tx_id.eq(tx_id))
+        .filter(tx_out::index.eq(tx_index))
```

Comment:
Changed the tables to tx_out which is not just containing unspent UTxO but also spent UTxO's it will return a wrong set of data. It is needed to take the data from utxo_view to get valid and up to date UTxO's. 
Like this you will query a lot of invalid (old / already spent) UTxOs which will lead to wrong transactions and information.
This should have been found during testing, this function is also used at other spots and the change breaks functionality.